aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -59,6 +59,18 @@ SignalR hub methods now support injecting services through dependency injection
 
 Hub constructors can accept services from DI as parameters, which can be stored in properties on the class for use in a hub method. For more information, see [Inject services into a hub](xref:signalr/hubs?view=aspnetcore-7.0&preserve-view=true#inject-services-into-a-hub)
 
+## Performance
+
+### Improved HTTP/2 performance when using many streams on a connection
```

Comment:
We made a later change that significantly rearchitected this code, so let's remove this section and replace it with:


> ### HTTP/2 Performance improvements
> 
> .NET 7 introduces a significant re-architecture of how Kestrel processes HTTP/2 requests. ASP.NET Core apps with busy HTTP/2 connections will experience reduced CPU usage and higher throughput.
> 
> Previously, the HTTP/2 multiplexing implementation relied on a lock controlling which request could write to the underlying TCP connection. A thread-safe queue replaces the write lock, so  instead of fighting over who gets to use the write lock, requests now queue up in an orderly line, and a dedicated consumer processes them. Previously wasted CPU resources are available to the rest of the app.
> 
> One place where these improvements can be noticed is in gRPC, a popular RPC framework that uses HTTP/2. Kestrel + gRPC benchmarks show a dramatic improvement:

![image](https://user-images.githubusercontent.com/219224/177910504-e93579b4-02e4-4079-8a8c-d9d24857aabf.png)

Reply:
cc: @JamesNK

Reply:
Ship it